clarifai/client/dataset.py

```diff
@@ -384,6 +387,61 @@ def upload_from_folder(self,
           folder_path=folder_path, dataset_id=self.id, labels=labels)
     self.input_object._bulk_upload(inputs=input_protos, batch_size=batch_size)
 
+  def get_upload_status(self, dataloader: Type[ClarifaiDataLoader],
```

Comment:
```suggestion
  def upload_and_get_status(self, dataloader: Type[ClarifaiDataLoader],
```

Reply:
`get_upload_status` fits better, this function only just gives statistics of upload process, upload is done via `upload_dataset()`.

Reply:
In what order would the user call these functions? Can we include the example usage in the docstring?
From this example it is not clear when a) `upload_dataset()` is called and b) why `get_upload_status()` is called on an empty dataset as it was just instantiated.
```
>>> from clarifai.client.dataset import Dataset
 >>> dataset = Dataset(dataset_id='dataset_id', user_id='user_id', app_id='app_id')
>>> dataset.get_upload_status(dataloader)
```

Reply:
Also, we could call this feature at the end of [upload_dataset()](https://github.com/Clarifai/clarifai-python/blob/68bcbd8890e19422d530144fa1feb7b48f4f056f/clarifai/client/dataset.py#L313). So by default user sees the stats after upload being done.

Reply:
Ya that would be great.